src/unofficial.md

```diff
@@ -31,7 +31,7 @@ Application domains:
 * [CXX — Safe Interop Between Rust and C++](https://cxx.rs)
 * [Triangle From Scratch](https://rust-tutorials.github.io/triangle-from-scratch/) - draw a triangle using Win32, but no external crates
 * [Writing Interpreters in Rust: a Guide](https://rust-hosted-langs.github.io/book/introduction.html)
-* [Using Unsafe for Fun and Profit](http://jakegoulding.com/rust-ffi-omnibus/) - FFI in depth
+* [The (unofficial) Rust FFI Guide](https://michael-f-bryan.github.io/rust-ffi-guide/) - FFI in depth
```

Comment:
I pasted the wrong URL by mistake (I shouldn't do such things in the middle of the night!). Thanks for correcting it.

I see now that the title of the book is "The (unofficial) Rust FFI Guide". The reason I thought it was "Using Unsafe for Fun and Profit" is that the book title doesn't appear in the default view unless you hover over it! See the screenshot below. I'm not a fan or disappearing UI elements, but there we are.
![Screenshot from 2021-08-31 12-24-22](https://user-images.githubusercontent.com/94953/131494549-6f3ab046-1d4f-4d3c-b752-c3485910ba76.png)